### PR TITLE
Tutorial

### PR DESCRIPTION
**Pull request by delta2323**
This PR improves the tutorial section in the document.

### CODE REVIEW

**Comment by delta2323** [comment]
As Jenkins has not been set up, I do not write doctest yet.

File: docs/source/tutorial/basic.rst [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+.. note::
+
+   The *with* statements in these codes are required to select the appropriate CUDA device.
+   If user uses only one device, these device switching is not needed.
+
+
+Data Transfer
+~~~~~~~~~~~~~
+
+:func:`cupy.cuda.to_gpu` is used to transfer data to GPU from either CPU (:class:`numpy.ndarray`)
```
Comment (delta2323) [inline docs/source/tutorial/basic.rst]: cupy does not have `to_gpu`

Comment (unnonouno) [inline docs/source/tutorial/basic.rst]: You need to use `cupy.ndarray(x_cpu)`?

Comment (delta2323) [inline docs/source/tutorial/basic.rst]: Yes, if we do not implement `to_gpu` and `to_cpu` to CuPy as well.

**Comment by delta2323** [comment]
I remove the section of how to write CPU/GPU agnostic code with CuPy because (so far as I understand), CuPy does not have a function equivalent to `chainer.cuda.get_array_module`. Conversely it is better to write the section if CuPy has the equivalent function.

**Comment by delta2323** [comment]
I restored CPU/GPU agnostic code section as there is `cupy.get_array_module` function.

**Comment by okuta** [comment]
Please use 4 spaces in test code.

**Comment by delta2323** [comment]
Thank you for your review. I fixed.

**Comment by okuta** [comment]
LGTM!

**Pull request merged**